lib/management/configuration.js

```diff
@@ -179,6 +179,8 @@ function patchConfiguration(newConf, log, cb) {
                         l.locationType });
                     return;
                 }
+                location.sizeLimitGB = l.sizeLimitGB || false;
```

Comment:
I'd expect `0 GB` to mean 'no limit' instead of `false GB` :)

Reply:
will fix to use undefined for 0 value case